docs/core/learn-more/authentication.md

```diff
@@ -1,35 +1,97 @@
-# Authentication
+# Authentication and Authorization
 
-## Introduction
+## Authentication
+
+Chain Core allows control of which entities have access to certain features of the system. There are two methods available for limiting access:
```

Comment:
It might be good to use the word "credential" in the second sentence, since it's a familiar term of art.